src/platform/android/NativeCodec.cpp

```diff
@@ -52,6 +57,13 @@ static jfieldID BitmapConfig_RGB_565;
 static jfieldID BitmapConfig_HARDWARE;
 
 void NativeCodec::JNIInit(JNIEnv* env) {
+  ColorSpaceClass = env->FindClass("android/graphics/ColorSpace");
+  ColorSpace_get =
+      env->GetStaticMethodID(ColorSpaceClass.get(), "get",
+                             "(Landroid/graphics/ColorSpace$Named;)Landroid/graphics/ColorSpace;");
+  ColorSpaceNamedClass = env->FindClass("android/graphics/ColorSpace$Named");
+  ColorSpaceNamed_SRGB = env->GetStaticFieldID(ColorSpaceNamedClass.get(), "SRGB",
+                                               "Landroid/graphics/ColorSpace$Named;");
```

Comment:
These interface calls also need to check for exceptions and clear them; otherwise, subsequent code will throw an exception.

Reply:
The ColorSpace class has been available since API level 26.

Reply:
Done~